v1.9.1 dashboard panic

## Bug Report

### Description

When running `talosctl dashboard` a panic was observed

### Logs

```
$ talosctl dashboard
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x78 pc=0x27f7cdd]

goroutine 148 [running]:
github.com/siderolabs/talos/internal/pkg/dashboard/apidata.(*Node).UpdateDiff(0xc00083e300, 0xc000fe8580)
	/src/internal/pkg/dashboard/apidata/node.go:182 +0x15d
github.com/siderolabs/talos/internal/pkg/dashboard/apidata.(*Data).CalculateDiff(0xc00016f140, 0xc000a187b0)
	/src/internal/pkg/dashboard/apidata/data.go:35 +0xe5
github.com/siderolabs/talos/internal/pkg/dashboard/apidata.(*Source).run(0xc000b0e200, 0xc00012aa80)
	/src/internal/pkg/dashboard/apidata/source.go:69 +0x10d
created by github.com/siderolabs/talos/internal/pkg/dashboard/apidata.(*Source).Run in goroutine 145
	/src/internal/pkg/dashboard/apidata/source.go:42 +0xd7
```

### Environment

- Talos version:

```
Client:
	Tag:         v1.9.1
	SHA:         348472f9
	Built:       
	Go version:  go1.23.4
	OS/Arch:     linux/amd64
Server:
	NODE:        100.111.83.104
	Tag:         v1.9.1
	SHA:         348472f9
	Built:       
	Go version:  go1.23.4
	OS/Arch:     linux/amd64
	Enabled:     RBAC
```

- Kubernetes version:
```
$ kubectl version
Client Version: v1.32.0
Kustomize Version: v5.5.0
Server Version: v1.32.0
```
- Platform:
```
Client: Ubuntu 24.04.1
Server: Hetzner VM
```

--- a/internal/pkg/dashboard/apidata/node.go
+++ b/internal/pkg/dashboard/apidata/node.go
@@ -179,8 +179,8 @@ func (node *Node) UpdateDiff(old *Node) {
 		node.SystemStatDiff = &machine.SystemStat{
 			// TODO: support other fields
 			CpuTotal:        cpuInfoDiff(old.SystemStat.GetCpuTotal(), node.SystemStat.GetCpuTotal()),
-			ContextSwitches: node.SystemStat.ContextSwitches - old.SystemStat.ContextSwitches,
-			ProcessCreated:  node.SystemStat.ProcessCreated - old.SystemStat.ProcessCreated,
+			ContextSwitches: node.SystemStat.GetContextSwitches() - old.SystemStat.GetContextSwitches(),
+			ProcessCreated:  node.SystemStat.GetProcessCreated() - old.SystemStat.GetProcessCreated(),
 		}
 	}
 

